contracts/counter/counter.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity >=0.5.11;
+
+contract Counter {
```

Comment:
can we delete this contract and use the one available on `tests-solidity/suites/basic/contract/Counter.sol`?